app/models/viewmodels/ApplicationsTab.scala

```diff
@@ -53,16 +53,21 @@ object ApplicationsTab {
 
   def assignedToMeCases(cases: Seq[Case]): ApplicationTabViewModel = {
 
-    val atars = cases.filter(aCase => aCase.application.isBTI && aCase.status == CaseStatus.OPEN)
+    val assignedCases =
+      cases.filter(aCase => aCase.status == CaseStatus.OPEN)
```

Comment:
Nice, maybe this initial filter of caseType could be repeated below for gateway cases, too?